qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -503,9 +503,8 @@ def _run_analysis_callback(
             return callback_id, True
         except Exception as ex:  # pylint: disable=broad-except
             self._analysis_callbacks[callback_id].status = AnalysisStatus.ERROR
-            error_msg = f"Analysis callback failed [Analysis ID: {callback_id}]:\n" "".join(
-                traceback.format_exception(type(ex), ex, ex.__traceback__)
-            )
+            tb_text = "".join(traceback.format_exception(type(ex), ex, ex.__traceback__))
+            error_msg = f"Analysis callback failed [Analysis ID: {callback_id}]:\n{tb_text}"
```

Comment:
just for cleanup?

Reply:
Yes, without this change, "Analysis callback failed..." gets prepended to every stack frame in the traceback which I don't think was the intent. Now it prints that message once and then prints the traceback normally.